scripts/jinja_gen.py

```diff
@@ -53,7 +55,7 @@ def read_jinja_parameters_from_file(filepath):
     parser.add_argument('--udp_onboard_gimbal_port_remote', default=13030, help="Mavlink Gimbal UDP for SITL")
     parser.add_argument('--generate_ros_models', default=False, dest='generate_ros_models', type=str2bool,
                     help="required if generating the agent for usage with ROS nodes, by default false")
-    parser.add_argument('--override_parameters_json_path', default='', help="json file with varaibles to override jinja parameters")
+    parser.add_argument('--override_parameters_json_path', default='/tmp/px4_gazebo_jinja_parameters_0.json', help="json file with variables to override jinja parameters")
```

Comment:
That's the question actually, in my case I generate this file in a docker container, and only then I thought it could be useful here. So, that s why I asked what would be the best location in comments.

Reply:
Maybe in `resources`? Probably set the default there, and if you want to override it with custom configs, you can specify the path